recipes/abseil/all/conandata.yml

```diff
@@ -2,3 +2,6 @@ sources:
   "20200205":
     sha256: 3c554df4909c5c55a6d251f6eadc2c78ff20db5ad4471fd9cbf8085c51b76797
     url: https://github.com/abseil/abseil-cpp/archive/08a7e7bf972c8451855a5022f2faf3d3655db015.tar.gz
+  "20200225.2":
```

Comment:
Abseil did delete their tags in the past, breaking [older versions from our Bincrafters' recipe for it](https://github.com/bincrafters/conan-abseil) and we have no guarantee that they won't do it again
Therefore, I would not be surprised if their delete their old LTS branches on day too

This started this discussion: https://github.com/conan-io/conan-center-index/issues/742

I would feel much more comfortable packaging commits from the master branch

Reply:
@Croydon  Thanks for reporting us! I agree with you, using commit will be better for Abseil.

Reply:
Add a comment in the `conandata.yml` file as a reminder for future versions.

Reply:
Done.